frontend/dashboard/components/RepoList/RepoList.tsx

```diff
@@ -1,311 +1,153 @@
-              icon={<i className={cn('fa fa-gitea', classes.linkIcon, classes.repoLink)} />}
-              key={`dashboard.repository${params.row.id}`}
-              label={t('dashboard.repository_in_list', { appName: repo })}
-              onClick={() => (window.location.href = params.row.html_url)}
-              showInMenu={false}
-              edge='end'
-            />,
-            <GridActionsCellItem
-              className={cn(classes.actionLink, classes.editLink)}
-              icon={
-                <PencilIcon
-                  title={t('dashboard.edit_app_icon')}
-                  className={cn(classes.linkIcon, classes.editLink)}
-                />
-              }
-              key={`dashboard.edit_app${params.row.id}`}
-              label={t('dashboard.edit_app', { appName: repo })}
-              onClick={() => (window.location.href = editUrl)}
-              showInMenu={false}
-            >
-              <a
-                key={params.row.id}
-                href={params.row.html_url}
-                className={cn(classes.actionLink, classes.repoLink)}
-              >
-                <span>{t(editTextKey)}</span>
-                <PencilIcon className={classes.linkIcon} />
-              </a>
-              ,
-            </GridActionsCellItem>,
-            <GridActionsCellItem
-              icon={<FilesIcon className={classes.dropdownIcon} />}
-              key={`dashboard.make_copy${params.row.id}`}
-              label={t('dashboard.make_copy')}
-              onClick={() => {
-                setModalOpen(true);
-                setCopyCurrentRepoName(repoFullName);
-              }}
-              showInMenu
-            />,
-            <GridActionsCellItem
-              icon={<ExternalLinkIcon className={classes.dropdownIcon} />}
-              key={`dashboard.open_in_new${params.row.id}`}
-              label={t('dashboard.open_in_new')}
-              onClick={() => window.open(editUrl, '_blank')}
-              showInMenu
-            />,
-          ];
-        },
-      },
-    ];
-
-    return [favouriteActionCol, ...columns, ...actionsCol];
-  }, [setStarredRepo, t, unsetStarredRepo]);
-
-  const handleCloseCopyModal = () => {
-    setModalOpen(false);
-    setCopyCurrentRepoName(null);
+  const paginationTexts: PaginationTexts = {
+    pageSizeLabel: t('dashboard.rows_per_page'),
+    totalRowsText: t('dashboard.total_count'),
+    nextButtonAriaLabel: t('general.next'),
+    previousButtonAriaLabel: t('general.previous'),
+    numberButtonAriaLabel: (number) => `${t('general.page')} ${number}`,
   };
 
-  const localText = {
-    ...nbNO.components.MuiDataGrid.defaultProps.localeText,
-    noRowsLabel: t('dashboard.no_repos_result'),
+  const paginationProps: RemotePaginationProps = {
+    currentPage: pageNumber + 1,
+    totalRows,
+    totalPages: Math.ceil(totalRows / pageSize),
+    pageSize,
+    pageSizeOptions,
+    onPageChange: (page: number) => onPageChange(page - 1),
```

Comment:
Could these index-adaption (+1/-1) be done directly in the view only?

Reply:
I'm thinking that we should keep using 1-indexes for currentPage and onPageChange in the component itself, so that  we are aligned with the [original Designsystem pagination component](https://storybook.designsystemet.no/?path=/docs/komponenter-pagination--docs) - just in case they want to use our component in the future. But I agree that we can move the index adaptations elsewhere, so I'll try moving them into the fetching hook instead 😊

Reply:
Found out that `mapQueryParams` had this index adaptation that was needed for the MUI component. When removing this, we don't need to adapt indexes in `RepoList` anymore 😊 
````
const mapQueryParams = (params: SearchRepoFilterParams): SearchRepoFilterParams => {
  const copyParams = { ...params };
  switch (params.sortby) {
    case 'name':
      copyParams.sortby = 'alpha';
      break;
    case 'updated_at':
      copyParams.sortby = 'updated';
      break;
  }
  copyParams.page = params.page + 1; // <--------- Has now been removed
  return copyParams;
};
````